.github/dependabot.yml

```diff
@@ -0,0 +1,17 @@
+version: 2
+updates:
+  - package-ecosystem: "bundler"
+    directory: "/"
+    target-branch: "develop"
+    schedule:
+      interval: "daily"
+    commit-message:
```

Comment:
on PR?

Reply:
@cbaudouinjr bump

Reply:
This isn't done on PRs, rather it's a job that executes daily on the develop branch.